### ISSUE TITLE
Single sample dataset

### ISSUE THREAD

**Issue by shashwatsahay** [SayakaMiura/ClonePhyTester#2]
Is it possible to create a simulated dataset with single samples to test modules like Fastclone https://github.com/GuanLab/FastClone_GuanLab

**Comment by SayakaMiura** [comment]
ClonePhyTester does not have a function to simulate datasets. However, if you like to use single samples, you can independently analyze each sample from a dataset. For example, for ClonePhyTester/G7/G7-100_readcount.txt dataset (seven samples, R1-R7):
SNVID	Wild	Mut	R2:ref	R2:alt	R3:ref	R3:alt	R4:ref	R4:alt	R6:ref	R6:alt	R1:ref	R1:alt	R5:ref	R5:alt	R7:ref	R7:alt
S1	C	A	75	30	93	22	54	50	51	52	78	22	77	30	32	39
S2	C	T	57	30	59	31	34	57	46	47	60	31	74	33	53	42
...
You can separate it into seven datasets:
G7-100_readcount_Sample_R2.txt
SNVID	Wild	Mut	R2:ref	R2:alt	
S1	C	A	75	30	
S2	C	T	57	30	
...
G7-100_readcount_Sample_R3.txt
SNVID	Wild	Mut	R3:ref	R3:alt	
S1	C	A	93	22	
S2	C	T	59	31	
...
And so on. Then, please adjust the format of the datasets for Fastclone.

**Comment by shashwatsahay** [comment]
Yeah, but we were thinking of how to get the true phylogenies in this case, so that we can test against others when only considering single dataset....

**Comment by SayakaMiura** [comment]
How about summing up all read counts, assuming that all the samples are sequenced together? Then, you can adjust the counts based on the sequence depths you like to use. In this way, the same true phylogenies can be used.

**Comment by shashwatsahay** [comment]
I have done this and the results look okayish, but I don't understand the logic behind this...
won't the software dedicated to working with multiple samples give out a better result???

**Comment by SayakaMiura** [comment]
Yes, theoretically, multiple samples should produce more accurate results than giving single samples. But, if a method produce accurate results from single samples, that is a great new as many data has only single samples per patient.

**Comment by shashwatsahay** [comment]
exactly and that's why we wanted to simulate a single sample dataset and then have a uniform dataset to test among different solvers...
we are currently looking at the data simulation method that pairtree describes to check if this can be done...
